src/components/base/NavBar/MainBlock.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import styled, { css } from 'styled-components'
+import { ReactComponent as EarthSvg } from '../../../svg/earth.svg'
+import { ReactComponent as BottomSvg } from '../../../svg/bottom.svg'
+import BaseButton from '../../common/BaseButton'
+    
+const HostWrapper = styled.div`
+    border-radius: 22px;
+    padding: 12px;
+`
+
+const LangCurr = styled.div`
+    background: transparent;
+    padding: 12px;
+    align-items: center;
+    display:flex;
+`
+
+const MainWrapper = styled.div`
+    height: 42px;
+    display:flex;
```

Comment:
```suggestion
    display: flex;
```
prettier를 사용하고 계신가요? 
eslint, prettier를 엮어서 사용해주시면 좋을 것 같습니다.